### PR TITLE
RecordMailer: clean up formatting

### PR DESCRIPTION
**Pull request by dunn**
## add an extra linebreak between records
Screen Shot 2019-07-10 at 6 55 38 PM

## remove array cruft from record title
Screen Shot 2019-07-10 at 6 55 45 PM

### CODE REVIEW

File: app/models/record_mailer.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def email_record(documents, details, url_gen_params)
             rescue
               I18n.t('blacklight.email.text.default_title')
             end
-    subject = I18n.t('blacklight.email.text.subject', count: documents.length, title: title)
+    subject = I18n.t('blacklight.email.text.subject', count: documents.length, title: title.first)
```
Comment (jkeck) [inline app/models/record_mailer.rb]: Is this guaranteed to be an array?

Comment (dunn) [reply app/models/record_mailer.rb]: I think yes, unless the user application overrides the title field to be single-valued?

Comment (jkeck) [reply app/models/record_mailer.rb]: Yeah, but they could do that theoretically do that I believe, yeah? (and with something like the canonical title of an item, that might be more common).

Comment (dunn) [reply app/models/record_mailer.rb]: Yeah, that's true.  I'll take another pass at this.

Comment (dunn) [reply app/models/record_mailer.rb]: `Array(title).first` should work.

**Pull request merged**